### PR TITLE
Meta: Rename “remove” to “hide” in all features

### LINKED ISSUE #4510: Rename “remove” to “hide” in all features
To be more consistent across features, we should probably choose one of these similar verbs. I think “hide” is more correct. In many cases we should also stop using .remove() and use .hidden = true instead, unless the CSS order matters.

Related: #4509

### PR DESCRIPTION
**Pull request by sahilcode17**
Fixes #4510 

Files which were renamed were as follows:
`remove-diff-signs` -> `hide-diff-signs`
`remove-label-faster` -> `hide-label-faster`

### CODE REVIEW

**Comment by fregante** [comment]
Please follow the PR template and use a proper PR title

**Comment by fregante** [comment]
Also migrations are needed for the features that have been renamed: https://github.com/sindresorhus/refined-github/blob/ac8de6bd0ef0a6016732a69556e4eb9c014cfd38/source/options-storage.ts#L28

**Comment by sahilcode17** [comment]
> Also migrations are needed for the features that have been renamed:
> 
> https://github.com/sindresorhus/refined-github/blob/ac8de6bd0ef0a6016732a69556e4eb9c014cfd38/source/options-storage.ts#L28

is there any other file where changes are to be done except options-storage.ts

**Comment by fregante** [comment]
No, migrations only exist there

File: source/features/hide-diff-signs.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.rgh-hide-diff-signs .blob-code-inner::before, /* Inline review blocks in PRs */
+.rgh-hide-diff-signs .blob-code-marker-cell::before { /* Commit page */
```
Comment (yakov116) [inline source/features/hide-diff-signs.css:2]: Not related to this PR but I think this feture is broken

**Comment by yakov116** [comment]
The readme must be updated

File: source/features/hide-diff-signs.css (hunk 1 above)
Comment (fregante) [reply source/features/hide-diff-signs.css:2]: Is it broken or is it #4695

**Comment by fregante** [comment]
> The readme must be updated

ugh it's unfortunate that https://github.com/sindresorhus/refined-github/pull/4495 meant that this is no longer caught by Webpack and would have been a runtime issue 😩

File: source/features/hide-diff-signs.css (hunk 1 above)
Comment (yakov116) [reply source/features/hide-diff-signs.css:2]: :shrug: I am not by a computer to check.

**Comment by sahilcode17** [comment]
not sure how to resolve the lint error :(

File: source/features/hide-diff-signs.css (hunk 1 above)
Comment (yakov116) [reply source/features/hide-diff-signs.css:2]: It is!

File: source/options-storage.ts [hunk 2]
```diff
@@ -26,6 +26,10 @@ const migrations = [
 	featureWasRenamed('remove-projects-tab', 'remove-unused-repo-tabs'), // Merged in July
 	featureWasRenamed('remove-unused-repo-tabs', 'clean-repo-tabs'), // Merged in July
 	featureWasRenamed('more-dropdown', 'clean-repo-tabs'), // Merged in July
+	featureWasRenamed('remove-diff-signs', 'hide-diff-signs'), // Merged in August
+	featureWasRenamed('remove-label-faster', 'hide-label-faster'), // Merged in August
+	
+
```
Comment (yakov116) [inline source/options-storage.ts:32]:
```suggestion
```

**Comment by yakov116** [comment]
> not sure how to resolve the lint error :(

I fixed it for you.

Please update the readme to match the names the you changed

**Comment by sahilcode17** [comment]
> > not sure how to resolve the lint error :(
> 
> I fixed it for you.
> 
> Please update the readme to match the names you changed

thanks, @yakov116, by the way, which readme are you talking about here the refined-Github readme or the pull request.

**Comment by yakov116** [comment]
> the refined-Github readme

👍

File: source/features/hide-diff-signs.css (hunk 1 above)
Comment (fregante) [reply source/features/hide-diff-signs.css:2]: smdu0s5y16i71

**Review by fregante — APPROVED** [review]

**Review by yakov116 — APPROVED** [review]

**Review by yakov116 — CHANGES_REQUESTED** [review]
Please follow the PR template

Please put into body of the issue what issue it closes and what files were renamed.

**Comment by sahilcode17** [comment]
> Fixes #4510
> 
> This pull request changed "remove" to "hide" in the file name in features.
> Also in the files imports were also changed accordingly.
> Files which were renamed were as follows:
> remove-diff-signs.tsx -> hide-diff-signs.tsx
> remove-diff-signs.css -> hide-diff-signs.css
> remove-label-faster.css -> hide-label-faster.css
> remove-label-faster.tsx -.> hide-label-faster.tsx
> along with this, some changes are also made in README.md file.

is it fine or something is missing?

**Review by yakov116 — APPROVED** [review]

**Comment by yakov116** [comment]
Thanks @sahilcode17!

**Comment by altrisi** [comment]
Doesn't `remove-label-faster` actually remove labels from the issue/pr, not only hide them? (it has been renamed -> `hide-label-faster` (here) -> `quick-label-hiding`)

**Comment by yakov116** [comment]
I think you correct. Maybe it should be `quickly-remove-issue-pr-labels`?

**Pull request merged**